src/profile-logic/process-profile.js

```diff
@@ -1222,7 +1222,7 @@ function sanitizeThreadPII(
         // Remove the URI fields from marker payload.
         // Mutating the payload here but it's safe because we copied the
         // markerTable already.
-        removeNetworkMarkerURLs(currentMarker);
+        markerTable.data[i] = removeNetworkMarkerURLs(currentMarker);
```

Comment:
you can remove the comment above now that we don't mutate the payload anymore

Reply:
Heh, and we saw that the comment was such a lie 😄